guardian/mobile-n10n
# Remove old sender worker CloudFormation file

## What does this change?

This PR removes the old `sender-worker-cfn.yaml` CloudFormation definition. I believe that the infrastructure for the worker lambdas is now defined via `cdk` (since https://github.com/guardian/mobile-n10n/pull/580) and as far as I can tell this file is [no longer used by Riff-Raff](https://github.com/guardian/mobile-n10n/blob/8cb80b95f3e191f429815e3755413633c98a13a6/notificationworkerlambda/riff-raff.yaml#L19) - see [here](https://github.com/guardian/mobile-n10n/pull/580/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91) and [here](https://github.com/guardian/mobile-n10n/pull/580/files#diff-b21a72a6aa116532f0737d074c7efc647b7f2d76a53c6420fb436e1b569d1911). I think the SQS queues defined in this stack/file became obsolete when work on https://github.com/guardian/mobile-n10n/pull/581 was completed and as far as I can tell all resources in this stack/file have now been deleted from AWS?

## How to test

I have [deployed this branch to `CODE`](https://riffraff.gutools.co.uk/deployment/view/96a63fcf-0c85-43eb-a09d-aa1be33b4d66) to confirm that the file is unused.

## How can we measure success?

There's less code/config in the repo and it's easier to see where the worker infrastructure is actually defined (the presence of this file confused me at first).

## Have we considered potential risks?

Yes, the biggest risk here is that I've misunderstood and that this file is still needed for something! Hopefully PR review & deploying to `CODE` is sufficient for mitigating that risk.
